packages/components/src/InputTime/InputTime.tsx

```diff
@@ -16,14 +16,23 @@ export function InputTime({
   ...params
 }: InputTimeProps) {
   const handleChange = (newValue: string) => {
+    if (newValue === "") {
+      return;
+    }
+
     onChange && onChange(htmlTimeToCivilTime(newValue));
   };
 
+  const formFieldValue = value ? civilTimeToHTMLTime(value) : undefined;
+
   if (supportsTime) {
     const fieldProps: FormFieldProps = {
       onChange: handleChange,
-      ...(defaultValue && { defaultValue: civilTimeToHTMLTime(defaultValue) }),
-      ...(value && { value: civilTimeToHTMLTime(value) }),
+      ...(value == undefined &&
+        defaultValue != undefined && {
+          defaultValue: civilTimeToHTMLTime(defaultValue),
+        }),
```

Comment:
The other error we were seeing, was if value wasn't defined, ex: "", we were trying to go back to defaultValue, but this was undefined.

Reply:
So now we only use defaultValue if value is NOT defined and defaultValue IS defined.